content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -7,11 +7,11 @@ description: Configure the kubelet's image credential provider plugin
 content_type: task
 ---
 
-{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
 <!-- overview -->
 
-Starting from Kubernetes v1.20, the kubelet can dynamically retrieve credentials for a container image registry
+Starting from Kubernetes *v1.20*, the kubelet can dynamically retrieve credentials for a container image registry
```

Comment:
just curious, why we are italicizing the versions?

Reply:
Note: Some of the changes in this PR were to accommodate comments from https://github.com/kubernetes/website/pull/37920/files

But I checked other docs, and saw the other docs are using unitalicized version numbers even for first occurrence, so I am following the same standard.